cli/src/main/package/setup.bat

```diff
@@ -8,6 +8,9 @@ Set _RESET=[0m
 pushd %~dp0
 echo Setting up IDEasy from %CD%
 
+REM activate ANSI support for colors in native cmd and powershell terminals
+reg add HKCU\Console /v VirtualTerminalLevel /t REG_DWORD /d 1 /f
```

Comment:
Very nice that you found this trick.
Should we add this to a `*.reg` file in `system/windows` like the other registry hooks?
Then also the MSI installer could apply the same change via that file.

Reply:
I moveed it into `.reg` files. One for enabling it and one for disabling it

Reply:
It is already defined in the IDEasy installation process which `.reg` files are executed?

Reply:
We had this in devonfw-ide but currently in IDEasy we only have optional `.reg` files.
Here is the devonfw-ide code:
https://github.com/devonfw/ide/blob/855ce79d795860416f73634397ea9a535c27a68c/scripts/src/main/resources/setup.bat#L22-L29

Reply:
done